Week 1/Weekly Practice Problems (Week 1)/1365: How Many Numbers Are Smaller Than the Current Number/raf.py

```diff
@@ -0,0 +1,7 @@
+        arr = []
```

Comment:
When we know the size of the list, it's better to initialize it in one go instead of  keep calling append (which will trigger buffer re-allocation at some point)
```suggestion
        arr = [None]*len(nums)
```